TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/Pipfile

```diff
@@ -13,6 +13,8 @@ sqlparse = "==0.5.0"
 pyopenssl = "*"
 
 [dev-packages]
+coverage = "*"
+moto = "*"
```

Comment:
It looks like you can install moto with just the specific dependencies you want to mock (https://docs.getmoto.org/en/latest/docs/getting_started.html#installing-moto) -- worth doing? I assume we're mocking a pretty narrow list